### ISSUE TITLE
WRN_ImplicitCopyInReadOnlyMember not given for event assignment

### ISSUE THREAD

**Issue by RikkiGibson** [dotnet/roslyn#35610]
```cs
using System;
public struct S {
    public event Action E;
    public readonly void M()
    {
        E += M2; // should warn about implicit copy
    }
    
    public static void M2() { }
}
```
